edb/common/ast/codegen.py

```diff
@@ -52,6 +52,15 @@ def node_visit(self, node: base.AST) -> None:
         visitor = getattr(self, method, self.generic_visit)
         return visitor(node)
 
+    def visit_indented(self, node: base.AST, indent: bool=False) -> None:
+        if indent:
+            self.new_lines = 1
+            self.char_indentation += 1
+        res = self.visit(node)
+        if indent:
+            self.char_indentation -= 1
+        return res
+
```

Comment:
I've added this helper that could probably simplify pg codegen a lot. If accepted here, I may do another refactor on it.

Reply:
Yeah that looks good. Should `indent` default to `True` here, though?